runtime/sydney/src/pos/mod.rs

```diff
@@ -90,12 +90,12 @@ parameter_types! {
 	pub const MaxCandidateIntake: u32 = 1;
 	pub const SocietyPalletId: PalletId = PalletId(*b"py/socie");
 
-	// Six sessions in an era (90 * 6 * 4 hours = 90 days eras).
-	pub const SessionsPerEra: SessionIndex = prod_or_fast!(90 * 6, 6);
+	// Six sessions in an era (6 * 4 hours = 1 day).
+	pub const SessionsPerEra: SessionIndex = prod_or_fast!(6, 6);
 
-	// 4 eras for unbonding (90 * 4 = 360 days).
-	pub const BondingDuration: sp_staking::EraIndex = prod_or_fast!(4, 360);
-	pub const SlashDeferDuration: sp_staking::EraIndex = BondingDuration::get() / 4;
+	// 360 eras for unbonding (1 * 360 = 360 days).
+	pub const BondingDuration: sp_staking::EraIndex = prod_or_fast!(360, 360);
+	pub const SlashDeferDuration: sp_staking::EraIndex = BondingDuration::get() / 360;
```

Comment:
`SlashDeferDuration` was 90 days in `prod` and `fast-runtime`(`BondingDuration::get() / 360`) when length of an era is 90 days in `prod` and 1 day in `fast-runtime`.(`pub const SessionsPerEra: SessionIndex = prod_or_fast!(90 * 6, 6)`)
I changed `SlashDeferDuration` to 1 day(`BondingDuration::get() / 360`) in both `prod` and `fast-runtime`.
Is this change right?

Reply:
That change is correct